lib/crypto-browser.js

```diff
@@ -0,0 +1,7 @@
+/*!
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
+ */
+// eslint-disable-next-line no-undef
+export const webcrypto = globalThis.crypto.webcrypto ?? globalThis.crypto;
```

Comment:
Do any browsers actually do `globalThis.crypto.webcrypto`? I thought this was node.js-specific. If we don't know of any browsers that do this, we should remove it. I would think that would also mean we could simplify the following line too.

Reply:
Ah yes, this comment in the [Node.js SDK for Web Crypto API](https://nodejs.org/api/webcrypto.html#web-crypto-api) suggests the same:

> Use globalThis.crypto or require('node:crypto').webcrypto to access this module.

I will simplify this code by reducing the entire code to one line:

```suggestion
export const CryptoKey = globalThis.CryptoKey ?? globalThis.crypto.CryptoKey;
```

Reply:
Actually, I also need to export `webcrypto`, so I will just change the first line to this:

```suggestion
export const webcrypto = globalThis.crypto;
```